### PR TITLE
Implement blockdiff check

### PR DESCRIPTION
**Pull request by raymondjacobson**

### CODE REVIEW

File: src/libs.ts [hunk 1]
```diff
@@ -6,7 +6,9 @@ const E = process.env
  * Singleton wrapper for Audius Libs.
  * Initialized in the server start-up.
  */
+// @ts-ignore - missing types on AudiusLibs
```
Comment (dylanjeffers) [inline src/libs.ts:9]: i think you can do `import type {AudiusLibs} from '@audius/sdk'`

File: src/servlets/api/index.ts [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
+const MIN_BLOCK_DIFFERENCE = 50
 
 export const router = express.Router()
 
 let usableDiscoveryProviders: string[] = []
 
 const updateDiscoveryProviders = async () => {
   const registeredVersion = await libs.ethContracts.getCurrentVersion('discovery-node')
+  console.info(LOG_PREFIX, `Registered version ${registeredVersion}`)
   let services = await libs.discoveryProvider.serviceSelector.findAll({ verbose: true })
```
Comment (dylanjeffers) [inline src/servlets/api/index.ts:20]: wondering if after typing libs this would type for you automatically too

**Review by dylanjeffers — APPROVED** [review]
nice change, lets possibly pair to see if the type will give you what you want

File: src/libs.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/libs.ts:9]: sozz

**Pull request merged**